libs/blocks/section-metadata/section-metadata.css

```diff
@@ -266,3 +266,38 @@ main > .section[class*='-up'] > .content {
     padding-right: var(--grid-margins-width-10);
   }
 }
+
+.section.masonry-layout {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: var(--spacing-s);
+  padding-left: var(--grid-margins-width);
+  padding-right: var(--grid-margins-width);
+}
+
+.section.masonry-layout > div:not([class]),
+.section.masonry-layout > div:not([class]) > div.fragment,
+.section.masonry-layout > div:not([class]) > div.fragment > div.section {
+  display: grid;
+}
+
+@media screen and (min-width: 768px) and (max-width: 1199px) {
```

Comment:
these are pretty specific `px` widths for the queries. Though I can guess the need for `max-width: 1199`, `768` seems awfully device specific.

Reply:
@JasonHowellSlavin As per the XD the grid layout needs to have a 2 column view only for 768px to 1199px viewport and for smaller tablet screen sizes and mobile screens it needs to be 1 column view

Reply:
@aishwaryamathuria  - It looks like the Figma file is wrong and doesn't use our set breakpoints. I'd talk to Kate and the consonant team and double check this. I'm w/ @JasonHowellSlavin on this one. I wouldn't recommend we use these additional breakpoints. I think as engineers we have to talk some of the designs w/ a grain of salt and resolve the discrepancies between design and reality.